authorization-common/src/main/java/com/sequenceiq/authorization/resource/RightUtils.java

```diff
@@ -0,0 +1,23 @@
+package com.sequenceiq.authorization.resource;
+
+import java.util.List;
+
+import com.google.common.collect.ImmutableList;
+
+public class RightUtils {
+
+    public static final String DISTROX_RIGHT_PREFIX = "datahub/";
+
+    static final List<AuthorizationResource> NON_DATAHUB_RESOURCE =
+            ImmutableList.of(AuthorizationResource.ENVIRONMENT, AuthorizationResource.FREEIPA, AuthorizationResource.DATALAKE);
+
+    private RightUtils() {
+    }
+
+    public static String getRight(AuthorizationResource resource, ResourceAction action) {
+        if (!NON_DATAHUB_RESOURCE.contains(resource)) {
+            return DISTROX_RIGHT_PREFIX + action.name().toLowerCase() + resource.getAuthorizationName();
```

Comment:
can we replace this ambiguous `name().lowerCase()` to a method on the enum so it would be more self describing and errorprune. thanks